package.json

```diff
@@ -308,9 +308,6 @@
     "webpack": "^4.0.1",
     "webpack-dev-middleware": "^3.0.0"
   },
-  "optionalDependencies": {
-    "fsevents": "*"
-  },
```

Comment:
@jwang1919 you originally put this in for Windows compatibility, but I don’t _think_ it should be necessary at this point. Can you see if you can still run the project on latest `master`? Thanks!

Reply:
Looks okay to remove the `fsevents`, but now running into a Python issue with `node-gyp`. More info here: https://github.com/popcodeorg/popcode/issues/1449